pom.xml

```diff
@@ -111,11 +102,25 @@
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
             <version>2.12.1</version>
+            <scope>provided</scope>
```

Comment:
what provides this dependency?
(I will note I cant actually remember what this lib is used for, but thats not important)

Reply:
this is actually googles implementation of json. But I think I can remove it now because I made my own implementation of json to not depend on anything else but the code in there.

Reply:
It's provided by minecraft. But as I've had in the past: they changed from snakeyaml 1 to snakeyaml 2... Which annoyed me a lot.

Reply:
👍 let me know when this one is good to approve and merge then